GitDiffMargin/View/PopupKeyboardBehavior.cs

```diff
@@ -69,7 +69,7 @@ private static void EnableKeyboardInput(Popup popup, bool enabled)
                 };
             };
 
-            popup.Closed += (sender, args) => Keyboard.Focus(element);
+            //popup.Closed += (sender, args) => Keyboard.Focus(element);
```

Comment:
❔ Can just delete?

Reply:
I regret that I didn't mention that I already tried just removing that line. Unfortunately, the focus does not go back to the `IWpfTextView` that the margin belonged to. Instead it focuses the `MainWindow` (top-level window of Visual Studio).

Reply:
@ricksladkey your answer is for the whole change or just when removing that line?

Reply:
@laurentkempe Sorry, I mistakenly assumed the change was just that one line because it was the only line that @sharwell commented on and so I thought the whole change was just deleting that line.  My mistake!

I will do a quick test against the whole PR...

Reply:
By the way, I had no idea that using the same local variable captured by two different closures would work, so I learned something by looking at that code!

Reply:
I will wait the answer from @MartinL0815 in #145 and if he doesn't reply in a week, I will then create a new release!

Reply:
Now that @MartinL0815 also tested it I will merge it!